### PR TITLE
opt-in show stats options

### PR DESCRIPTION
**Pull request by rjoydip**
PR for #287 #327 

Flag: **show_stats=forks**   **Default: null**

tempsnip

### CODE REVIEW

**Comment by WIttyJudge** [comment]
🙄🙄🙄

**Comment by anuraghazra** [comment]
Hmm? what is this count?

**Comment by rjoydip** [comment]
Number of repository forked by the user.

**Comment by rjoydip** [comment]
> Please update the Readme to include the "fork" hide option.

Done. Update `readme`.

**Comment by anuraghazra** [comment]
@rjoydip thanks for the PR, the only problem i have with adding more stats is that i don't want to bloat the card with stats and make it longer, so it might be good if we can make them Opt-In instead of Opt-Out

File: api/index.js [hunk 1]
```diff
@@ -61,6 +62,7 @@ module.exports = async (req, res) => {
       hide_title: parseBoolean(hide_title),
       hide_border: parseBoolean(hide_border),
       hide_rank: parseBoolean(hide_rank),
+      show_forks: parseBoolean(show_forks),
```
Comment (anuraghazra) [inline api/index.js:65]: This is not scalable. we have to do something like: show_stats=forks,extra,extra

Comment (rjoydip) [reply api/index.js:65]: Okay

Comment (rjoydip) [reply api/index.js]: done

**Comment by rjoydip** [comment]
removed followers flags https://github.com/anuraghazra/github-readme-stats/issues/327#issuecomment-670471927

**Comment by Xunzhuo** [comment]
why this PR is not merged yet? I think fork status is an important data as well

**Pull request closed without merging**